BTMLColorLOSMod/WeaponRangeIndicators_DrawLine_Patch.cs

```diff
@@ -162,15 +163,26 @@ public static class WeaponRangeIndicators_DrawLine_Patch
                                 Vector3 collisionPoint = previewInfo.collisionPoint;
                                 collisionPoint = Vector3.Project(collisionPoint - vector, vector2 - vector) + vector;
                                 line.SetPosition(1, collisionPoint);
+                                if(BTMLColorLOSMod.ModSettings.BlockedLOFLine1Active)
```

Comment:
do we need this setting?

Reply:
I figured some would want the first half of the line red, the second half a different color; and some would want both lines different colors. This just preserves the original code path if you don't want the first half changed.

I don't feel strongly.